commercial/app/commercial/CommercialLifecycle.scala

```diff
@@ -23,24 +33,15 @@ trait CommercialLifecycle extends GlobalSettings with Logging with ExecutionCont
     MoneyBestBuysRefresh
   )
 
-
-  private lazy val metricMap = Map(
-    "fetch-failure" -> AkkaAgent(0.0),
-    "fetch-success" -> AkkaAgent(0.0),
-    "parse-failure" -> AkkaAgent(0.0),
-    "parse-success" -> AkkaAgent(0.0)
-  )
-
-  private def recordEvent(eventName:String, outcome:String):Unit = {
+  private def recordEvent(eventName:String, outcome:String): Unit = {
     val keyName = s"$eventName-$outcome"
-    metricMap
+    CommercialLifecycleMetrics.metricMap
       .get(keyName)
       .foreach(agent => agent.send(_ +1.0))
   }
 
-
   private def updateMetrics(): Unit = {
-    metricMap.foreach{
+    CommercialLifecycleMetrics.metricMap.foreach{
```

Comment:
Just curious: why not move the whole `recordEvent` and `updateMetrics` methods to the `CommercialLifecycleMetrics`, so you keep the map of agents private and not expose it?